phpBB/styles/prosilver/theme/buttons.css

```diff
@@ -20,6 +20,7 @@
 	-moz-user-select: none;
 	-ms-user-select: none;
 	user-select: none;
+	moz-border-radius: 4px; -webkit-border-radius: 4px; border-radius: 4px;
```

Comment:
This is redundant code

Reply:
also,
firstly when submitting prs please follow the guidelines for commits https://area51.phpbb.com/docs/dev/3.2.x/development/git.html
second please be sure any changes adhere to our coding standards for 3.3 which are currently in draft and can be found here: https://github.com/phpbb/documentation/pull/128
also, our supported browsers are listed here: https://github.com/phpbb/phpbb/blob/master/gulpfile.js#L23

Reply:
Thanks for the guiding. Will check them out.

Reply:
also you have no ticket for this https://tracker.phpbb.com/secure/Dashboard.jspa